change name of AdvisorTextBubble, AdvisorPanel, Advisor

drsmall opened the pull request:
Advisor->Adviser
AdvisorTextBubble->Adivser
AdvisorPanel->AdviserList

rraub commented:
`/client/components/game/adviser-list/tests/advisor-panel.jsx` still needs to be renamed and the linter is complaining about that.

Commit:
Made Changes To Show Main Page.

Commit:
Put In throw just before game. Plz look at branch and look at game. Before
we meet (via internet phone)

Commit:
Adding In Crash

Commit:
Fixed Bug Related To UI

Commit:
Merge branch 'develop' of https://github.com/gios-asu/dont-get-fired-game into WeekendPUSH
Fixed UI Components. Game Page now renders correctly.

Commit:
Created New BackEnd Files Called CS-... Forming them to the pages, and
going to warp the events around them. You can see programs, and buy them.
Game is more functional.

xyan12345 committed:
add cs-program data

xyan12345 committed:
modify program data

xyan12345 committed:
Modified Program Json

xyan12345 committed:
Modified Data Program

drsmall committed:
Added randomization for game settins

Commit:
Broke 10 Tests. But Nearly all parts work.
Advisors Logic needs fine tuned,
Final Page Needs Fine Tuned
This will be our Thusday Starting Place. 7 Days Left, so pushing anyway.

xyan12345 committed:
Fixed part of issue...

xyan12345 committed:
Fix issue

rraub committed:
Merge pull request #128 from gios-asu/xuefeng-develop

Xuefeng develop

drsmall committed:
Fix linting issues

drsmall committed:
Merge branch 'WeekendPUSH' of http://github.com/gios-asu/dont-get-fired-game into david-capstone-develop
